### PR TITLE
set up initial dev tools

### PR DESCRIPTION
**Pull request by brad90four**
This commit will add the initial development tools to use for PyWeek33
Adds:
-pre-commit
-flake8
-poetry
-git message template

### CODE REVIEW

File: .gitmessage [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# 50 characters                                 #
+# (When implemented this commit will ___)
+
+# 72 characters                                                       #
```
Comment (Kronifer) [inline .gitmessage:4]: Whats with the random hashtags?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,9 @@
 # pyweek-33
 The official repo for Cat Dev PyWeek 33 shenanigans!
+
+
+## Setup
+1. Clone the repo: `git clone https://github.com/cat-dev-group/pyweek-33.git`
+2. Install dependencies from `poetry` via: `poetry install`
+3. Set up pre-commit hooks: `pre-commit install`
+4. Add commit message template: `git config commit.template .gitmessage`
```
Comment (Kronifer) [inline README.md:9]:
```suggestion
4. Add commit message template: `git config --local commit.template .gitmessage`
```

**Review by Kronifer — CHANGES_REQUESTED** [review]
Few nitpicks, and I imagine that the pyproject.toml will be updated as we go.

File: .gitmessage (hunk 1 above)
Comment (brad90four) [reply .gitmessage:4]: The ones at the end of the line are to show where the 50 and 72 character limits would be. The subject can be up to 50 before getting the `...` in the preview, and GitHub wraps the body of commit messages at 72 chars

Comment (Kronifer) [reply .gitmessage:4]: Thanks for the explanation!

**Review by Kronifer — APPROVED** [review]
Lgtm

**Review by Evorage0 — APPROVED** [review]
lgtm

**Pull request merged**